en/docs/develop-components/work-with-the-micro-integrator-runtime-in-choreo.md

```diff
@@ -42,7 +42,7 @@ Understanding the Integration Project directory structure is crucial for organiz
 
 ## OpenAPI support
 
-OpenAPI Support plays a crucial role in enhancing the functionality of integrations exposed as APIs. By providing an OpenAPI definition for your exposed integration, you can streamline testing and management processes, ensuring a more efficient and user-friendly experience.
+OpenAPI Support plays a crucial role in enhancing the functionality of integrations exposed as APIs. By providing an OpenAPI definition for your exposed integration, you can streamline testing and management processes, ensuring a more efficient and user-friendly experience. If you don't provide an OpenAPI definition, our system will automatically generate one for you, saving you time and effort.
```

Comment:
This creates an OpenAPI definition based on what? Also, we will need to include this info in the above section where it is mentioned that OpenAPI is an optional requirement.